docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+nfs:
+    image: misterbisson/nfsserver
+    privileged: true
+    mem_limit: 256m
+    restart: always
+    labels:
+      - triton.cns.services=nfs
+    expose:
+      - 111
+      - 1892
+      - 2049
+    environment:
+      - CONSUL=consul
```

Comment:
This should be `- CONSUL` unless you're linking; you'll only want `- CONSUL=consul` for the `local-docker-compose.yml`.